front/src/pages/SummaryPage/components/Form/styles.tsx

```diff
@@ -0,0 +1,119 @@
+import styled, { css } from "styled-components";
+
+import { theme } from "assets/theme";
+
+const { colors, media } = theme;
+
+const btn = (light: string, dark: string) => css`
+  white-space: nowrap;
+  display: inline-block;
+  border-radius: 5px;
+  padding: 5px 15px;
+  font-size: 16px;
+  color: white;
+  &:visited {
+    color: white;
+  }
+  background-image: linear-gradient(${light}, ${dark});
+  border: 1px solid ${dark};
+  &:hover {
+    background-image: linear-gradient(${light}, ${dark});
+    &[disabled] {
+      background-image: linear-gradient(${light}, ${dark});
+    }
+  }
+  &:visited {
+    color: black;
+  }
+  &[disabled] {
+    opacity: 0.6;
+    cursor: not-allowed;
+  }
+`;
+
+const btnDefault = css`
+  ${btn("#ffffff", "#d5d5d5")} color: #555;
+`;
+
+const btnPrimary = btn("#4f93ce", "#285f8f");
+
+export default styled.div`
+  form {
+    max-width: 500px;
+    margin: 10px auto;
+    border: 1px solid #ccc;
+    padding: 20px;
+    ${media.sm} {
+      padding: 0;
+    }
+    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.3);
+    border-radius: 3px;
+
+    & > div {
+      display: flex;
+      flex-flow: row nowrap;
+
+      line-height: 2em;
+      margin: 5px;
+      & > label {
```

Comment:
& > label
to to samo co
> label
nie musisz w tym przypadku dawać &

Reply:
Done.